docs/content/FAQ/_index.md

```diff
@@ -36,19 +38,27 @@ If you want to see what has changed between versions, refer to the [releases pag
 
 The Infection Monkey agent shuts off either when it can't find new victims or it has exceeded the quota of victims as defined in the configuration.
 
+## Where does Infection Monkey store runtime artifacts?
+
+The Infection Monkey agent may need to access or create some files on systems while running, such as during the execution of custom post-breach actions or when attempting the Zerologon exploit.
```

Comment:
This should be 2 sections. One for monkey island, and one for the agent. Zerologon doesn't store anything in `data_dir`

Reply:
2 sections? What's different between them? There's only one data directory?

Reply:
Zerologon is run by the monkey agent. There is no `data_dir` for monkey agent. Monkey agent may be running on machines that are not monkey island. Zerologon creates a temporary directory and stores its data there. Monkey agent also stores binaries and logs in a temporary directory.

Reply:
Oh, got it. I wasn't thinking about the agent. :sweat_smile: